app/controllers/reservations_controller.rb

```diff
@@ -2,10 +2,17 @@
 
 class ReservationsController < ApplicationController
   before_action :set_screening, only: %i[new create]
+  before_action :set_reservation, only: %i[edit update destroy]
+  def index
+    @reservations = Reservation.all
```

Comment:
In the views you refer to associated screenings movies and halls, this will create a lot of queries, 
make sure to include every associated model used in the views to avoid N+1 queries